system/Database/Forge.php

```diff
@@ -466,6 +466,8 @@ public function createTable(string $table, bool $ifNotExists = false, array $att
 
                 return false;
             }
+
+            return true;
```

Comment:
Is this a required change only for Oracle drivers?

Reply:
You're right!
I created a PR.
However, the only pattern where the `createTableIfStr` property is false is for SQLite less than 3.3, except for OCI8.
Therefore, it may be easier to maintain if you fix it in this PR as the test code is a little difficult to convey the intention as shown below.
Maybe I'm just a bad test code.

https://github.com/codeigniter4/CodeIgniter4/blob/f564f6b7eca2504ac894bbaf7c3b6d18ce9b7b85/tests/system/Database/Forge/CreateTableTest.php#L23-L40

Reply:
#5133 was merged. This change is no longer necessary.

Reply:
This change was merged into CI, so I rebased it.